### ISSUE TITLE
As we add new terms, should I submit NTRs?  From today, SimulConsult native code hresko_230208214543 UMLS C0340222

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#9612]
**Preferred term label:**
Tracheal rupture

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Ehlers Danlos

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
It would be very beneficial for the HPO to have term requests such as this!
I have added this term as
HP:0034531
def: A break or tear in the trachea.
and am thinking of for instance, https://pubmed.ncbi.nlm.nih.gov/20693885/
